### PR TITLE
Create ProjectBoard.md

### PR DESCRIPTION
**Pull request by its-a-lisa**
Tried to capture as much as possible our workflow

### CODE REVIEW

File: ProjectBoard.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+> 
+> **User Story:**
+> 
+> **Implementation:**
+> * [ ] [first thing to do]
+> * [ ] [another thing to do]
+>
+> **Acceptance Criteria:**
+##Boards
+### Cloumns
```
Comment (JJediny) [inline ProjectBoard.md]: Minor format ^ and typo

File: ProjectBoard.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Determine which stories should be targeted for completion in the upcoming sprint by adding them to a sprint backlog list. 
+
+### Retros
+An opportunity for the team to inspect itself and create a plan for improvements to be enacted during the next Sprint.
+
+### Sprint Review
+A chance to look back and see what has been done and to communicate that with stakeholders. 
+
+## Epics
+TBD
```
Comment (JJediny) [inline ProjectBoard.md]: Can reserve but feel like we've for now agreed to create 1 `issue` per epic and create a label based off of it for association.

File: ProjectBoard.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- Use whatever labels works for you!
+  - John plans on using labels related to the Cybersecurity Framework 
+    - This helps him with tracking progress 
+  - Alyssa plans on using labels related to the domain of work that fits her skills 
+    - This helps her prioritize, sort & pull from
+
+### Assignments
+- If you would like to work on an issue, assign yourself to it before it gets to the ready column
+
+### Card template:
```
Comment (JJediny) [inline ProjectBoard.md]: Good for now but should prob create an issue to create templated issues via `.github/ISSUE_TEMPLATE`

- https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates
- Example: https://github.com/GSA/datagov-deploy/tree/develop/.github/ISSUE_TEMPLATE

File: ProjectBoard.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+> **Acceptance Criteria:**
+##Boards
+### Cloumns 
+**Definition of "Done"**
+
+An agile "Definition of Done" (DoD) captures the team's agreed-upon standards for how we get work done at a consistent level of quality. Having a DoD ensures that non-functional requirements (NFRs) don't have to be re-litigated for every piece of work taken on, cards can be focused on just the relevant details, and new team members aren't surprised by assumed expectations of their colleagues.
+
+At our sprint reviews, we celebrate work that has reached the "Done" column and is of interest to our users, teammates, or other people apart from the squad that built it. 
+
+**Column exit criteria**
```
Comment (JJediny) [inline ProjectBoard.md]: :100:

File: ProjectBoard.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+Before advancing a card from one column to the next on the board, it should meet the "exit criteria" for the current column, which are listed below.
+
+- New Issues (where reported issues, and newly-identified stories appear)
+- Icebox (where they go when they are not planned to get attention any time soon)
+- Grooming (when they're being refined for implementation)
+- Backlog/Ready (when they are sequenced alongside other stories prioritized for attention)
+- In Progress (when someone is actively working on the issue)
+- Blocked/waiting (when there's a dependency on someone responding or something happening outside the team)
+- Awaiting Acceptance (when work is considered complete and awaiting review/merging/feedback)
```
Comment (JJediny) [inline ProjectBoard.md]: Dont have this on our board suggest removing for now

File: ProjectBoard.md (hunk 3 above)
Comment (its-a-lisa) [reply ProjectBoard.md]: Agree

File: ProjectBoard.md (hunk 2 above)
Comment (its-a-lisa) [reply ProjectBoard.md]: Ok, I edited it to reflect this. I knew we had mentioned moving things to a word doc and creating roadmaps, and honestly by time I got to this I was spent, so really appreciate the details to fill in.

File: ProjectBoard.md (hunk 5 above)
Comment (its-a-lisa) [reply ProjectBoard.md]: Don't have the 'awaiting acceptance'? I put it on our board when I was doing this, but we can remove it for now and see if we're fine w/o it. I felt this past sprint that there were some things in the In Progress column that we were waiting on a step outside of the tasks we were responsible for, so it was taking up space. I can remove it for now and we can add it back in after a retro if its seen as helpful.

**Pull request merged**